addon/system/scroll-container.js

```diff
@@ -13,8 +13,8 @@ export default class ScrollContainer {
       let { top, left } = this.element.getBoundingClientRect();
       this.top = top;
       this.left = left;
-      this.width = getComputedStyle(this.element).width;
-      this.height = getComputedStyle(this.element).height;
+      this.width = parseFloat(getComputedStyle(this.element).width, 10);
+      this.height = parseFloat(getComputedStyle(this.element).height, 10);
```

Comment:
`parseFloat` doesn't have a 2nd param. It doesn't need a radix unlike `parseInt`.

Reply:
Done ✅